zp-relayer/workers/poolTxWorker.ts

```diff
@@ -133,7 +144,14 @@ export async function createPoolTxWorker({ redis, mutex, txManager, validateTx,
         processResult = await buildDirectDeposits(tx, treeProver, pool.optimisticState)
       } else if (type === WorkerTxType.Normal) {
         const tx = payload as WorkerTx<WorkerTxType.Normal>
-        await validateTx(tx, pool, traceId)
+
+        const requiredFee = await feeManager.estimateFee({
+          gasLimit: config.relayerGasLimit,
+          extraData: tx.rawMemo + (tx.depositSignature || ''),
```

Comment:
What about other mandatory parts of memo, like withdrawal addresses or permit expiry timestamps? Shouldn't we count them here as well?